### PR TITLE
Code review of the TSE

### PR DESCRIPTION
**Pull request by carloshdelreal**
project complete, the grey scale heatmap and the dummy text

### CODE REVIEW

**Comment by Torres-ssf** [comment]
Hi there team...
You got most of the things right. But before I give this project a 👍, we need to make some changes.
- [x] All content should be wrapped with `<main>` HTML tags
- [x] Every children of main should be a `<section>` HTML tag pair
- [x] Every children of section should be an `<article>` HTML tag pair

- [x] Your page must use the `<nav>` for the navbar. The navbar must have 7 separate containers (div is recommended) representing the icons and links.

- [x]  The four articles with 50% width (the main articles) must have an image in top left corner (like the original page) with the title of an article under it. Use a small container (div) to represent the image.

- [x] Three articles with a width of 33%.

- [x] Kindly create some links for the footer. They should cover the central part of the footer, both horizontally and vertically. It should have an unorganized list `<ul>`

- [ ] Fix the live version of your page.

Please kindly make these changes and submit for code review again.

File: index.html [hunk 1]
```diff
@@ -5,27 +5,326 @@
...
-            <div class="header-container">
-                <figure>
-
-                </figure>
-                <div></div>
-                <div></div>
-                <div></div>
+    <div class="main-wrapper">
+        <div class="header">
+            <div class="nav-header">
```
Comment (Torres-ssf) [inline index.html]: Use a `nav` with 7 containers inside it.

File: index.html [hunk 2]
```diff
@@ -5,27 +5,326 @@
...
-                <div></div>
-                <a href=""></a>
             </div>
-        </header>
-        <section>
-            <section class="left-column"></section>
-            <section class="right-column"></section>
-        </section>
+        </div>
+        <div class="section">
```
Comment (Torres-ssf) [inline index.html]: Each one of the 4 main articles should have an image in the top left corner with title of an article under it.

File: index.html [hunk 3]
```diff
@@ -5,27 +5,326 @@
...
+        <div class="col-1-4">
+            <h3>Lorem ipsum dolor sit amet consectetur adipisicing elit</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Sit, est sequi. Voluptate maxime optio ipsa consequuntur. Ut architecto ipsam molestias unde, recusandae quaerat fuga nemo distinctio sapiente natus accusantium pariatur!</p>
+        </div>
+        <div class="col-1-4">
+            <h3>Lorem ipsum dolor sit amet consectetur adipisicing elit</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Sit, est sequi. Voluptate maxime optio ipsa consequuntur. Ut architecto ipsam molestias unde, recusandae quaerat fuga nemo distinctio sapiente natus accusantium pariatur!</p>
+        </div>
+        <div class="col-1-4">
+            <div>
```
Comment (Torres-ssf) [inline index.html]: Your page must have a section with 3 articles of 33% width each. Delete one of these columns and give the 3 articles the same width.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]

**Comment by carloshdelreal** [comment]
___Fix the live version of your page.___ @Torres-ssf
Answer:
Once the development branch be merged with the master branch, then you will see the changes in the live page, that is because github pages only can be activated for the master branch

**Review by bellom — CHANGES_REQUESTED** [review]
Hi @carloshdelreal @ViriCruz, 
It not necessary you make use of gh-pages before you could create live link, kindly checkout this [video tutorials](https://www.loom.com/share/f7688bd6dbb54ae2a4e80a621fa7665d)

I can see you did a good job doing all the corrections pointed out from previous review but you misunderstood one and you left one out.
1. *You misunderstood :point_right:*  ` All content should be wrapped with <main> HTML tags`.
   What the reviewer is telling you is: navbar should be inside `nav tag` and page content inside `main tag`, read about best HTML5 practice [here](https://www.themelocation.com/best-html5-practices/)

2. *Leftout:* Reviewer made mention that the navbar must have 7 separate containers (div is recommended) representing the icons and links.

Kindly do these two things and your :+1: await you and submit for code review again.

**Comment by carloshdelreal** [comment]
We have already added the 7 containers to the navbar and distribute the semantic elements as is suggested in the best HTML5 practice [here](https://www.themelocation.com/best-html5-practices/)

Screenshot 2019-06-20 13 14 54

**Review by bellom — COMMENTED** [review]
Hi @carloshdelreal @ViriCruz,
The tags elements are now looking good.

image

My suggestion would be to improve the navbar with 7 divs, and also work on the UI, all the projects will be included in your portfolio.
checkout an example [here](https://nazeh.github.io/design-teardown/)
Goodluck in the next project....

**Comment by bellom** [comment]
Awesome, you don't need to submit another code review go ahead and merge to master and also fill the project completion form .

**Pull request merged**